IOS_CREDENTIALS_EAS.md

```diff
@@ -64,10 +90,10 @@ In your [Apple developer console](https://developer.apple.com) create two identi
 #### Note on App Groups at Build Time:
 Make sure to build with both identifiers containing the AppGroup simultaneously. To troubleshoot, remove them both and then rebuild with both identifiers' AppGroup capabilities enabled.
 
-## 3. Provisioning
+### 3. Provisioning
 Create AdHoc (local development) and AppStore (production) provisioning profiles with both identifiers (four provisioning profiles total). Both provisioning profiles should use the same distribution certificate that is used by your app. Download the profiles.
```

Comment:
It would be usefull to add info that if you are modifying anything in app identifier you need to recreate the provisioning profile